.github/scripts/check-for-unreleased-tags.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#Script to check tags against releases
+# check-for-unreleased-tags.sh --days DAYS
+set -e
+
+echo "=== Checking Git tags against GitHub releases (last 7 days) ==="
+echo ""
+
+# Calculate date 7 days ago
+SEVEN_DAYS_AGO=$(date -u -d '7 days ago' '+%Y-%m-%d %H:%M:%S')
+SEVEN_DAYS_AGO_UNIX=$(date -u -d '7 days ago' '+%s')
+echo "Checking tags and releases created since: $SEVEN_DAYS_AGO UTC"
+echo ""
```

Comment:
Thanks for pulling this out to its own script! But it 7 days is still a hard-coded value.

Per my earlier review, can we make this a configurable value and set the default in the CI workflow to 30 days?